azure-devops/azext_devops/dev/pipelines/pipeline.py

```diff
@@ -87,17 +90,19 @@ def pipeline_show(id=None, name=None, open=False, organization=None, project=Non
     client = get_build_client(organization)
     if id is None:
         if name is not None:
-            id = get_definition_id_from_name(name, client, project, path=folder_path)
+            id = get_definition_id_from_name(
+                name, client, project, path=folder_path)
         else:
-            raise CLIError("Either the --id argument or the --name argument must be supplied for this command.")
+            raise CLIError(
+                "Either the --id argument or the --name argument must be supplied for this command.")
     build_definition = client.get_definition(definition_id=id, project=project)
     if open:
         _open_pipeline(build_definition, organization)
     return build_definition
 
 
 def pipeline_run(id=None, branch=None, commit_id=None, name=None, open=False, variables=None,  # pylint: disable=redefined-builtin
-                 folder_path=None, organization=None, project=None, detect=None):
+                 folder_path=None, organization=None, project=None, detect=None, parameters=None):
```

Comment:
is `variables` and `parameters` basically the same thing ?

Reply:
**variable** is Key-value pair (separated by space)
**parameters** is a json structure (only single quote instead of double quote)